symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java

```diff
@@ -192,4 +201,26 @@ private void configureProxy(BdkProxyConfig proxyConfig, ApiClientBuilder apiClie
           .withProxyCredentials(proxyConfig.getUsername(), proxyConfig.getPassword());
     }
   }
+
+  @API(status = API.Status.INTERNAL)
+  public enum ServiceEnum {
+    AGENT,
+    KEY_MANAGER,
+    POD,
+    SESSION_AUTH
+  }
+
+  public static ServiceEnum getServiceNameFromBasePath(String basePath) {
+    if(basePath.contains("/relay") || basePath.contains("/keyauth")){
+      return ServiceEnum.KEY_MANAGER;
+    }
+    if (basePath.contains("/sessionauth")){
+      return ServiceEnum.SESSION_AUTH;
+    }
+    if(basePath.contains("/agent")) {
+      return ServiceEnum.AGENT;
+    }
+    else return ServiceEnum.POD;
```

Comment:
[minor] `if (` and `if(` are not consistent

Reply:
fixing it